### ISSUE TITLE
Link to IRC/Matrix room

### ISSUE THREAD

**Issue by Evidlo** [reHackable/awesome-reMarkable#19]
It would be nice if there were a centralized place to chat about the reMarkable and reMarkable development.  The reMarkable subreddit has a Discord, but that doesn't jive with FOSS.

I propose the creation of an IRC room (e.g. ##remarkable on Freenode) to encourage discussion surrounding the reMarkable and reMarkable development.

**Comment by CTXz** [comment]
I entirely agree that an IRC channel would be a great idea, however, for the same reason that we are not promoting the discord chat, nor the subreddit here, I would not promote an IRC chat channel here.

This repository is only a collective list of projects, not a wiki page.

@Ameb , your opinion?

**Comment by Ameb** [comment]
I agree that a Riot/IRC/Matrix (even gitter) room/server would align better with the FOSS nature of the reMarkable community.
A few days ago i saw the (probably yours) post on reddit about Matrix. It was too short/hard to understand and users flagged it as spam.

The best thing of the current Discord is that some members of the reMarkable team are there, @patrickhisnibrataas and @sandsmark. Having them reading us is a great thing.

There are many discussions on the internet already on the topic of discord vs irc. The proper tool imho is the one most people would use, or the one with some kind of support (rM devs there).

I don't see the need for another communication channel, unless the rM team creates an official one (forums as example). The proper place for this discussion might be Discord.

**Comment by CTXz** [comment]
Alright, I'll close the issue now, since this topic doesn't have much to do with the awesome list itself. For further discussion, head out to discord or reddit.
